index.html

```diff
@@ -2580,6 +2618,11 @@ <h3>
 that decision.
         </p>
 
+        <p>
+          Consumers SHOULD drop all properties from a DID Documents that 
+          are not defined by the <code>@context</code>. 
```

Comment:
Could we modify this to "Consumers SHOULD drop all properties from DID Documents that are not defined by `@context` or that they do not understand."?

Reply:
I cannot test for "do not understand" I can test for absence of properties in the context.

I assume that changing this to a MUST, would draw an even large distinction between JSON and JSON-LD Consumption. Given that a consumer can request content as `application/did+json` if they want nothing dropped, I'd be comfortable with a MUST here as well.

I would also be comfortable saying no properties are dropped and `@context` is required, and erasing the distinction between JSON-LD and JSON-LD with unknown properties :)